### PR TITLE
update PR template

### PR DESCRIPTION
**Pull request by jeff-dude**
proposed additions to PR template

### CODE REVIEW

**Comment by jeff-dude** [comment]
fyi @duneanalytics/team-data-experience

**Review by antonio-mendes — APPROVED** [review]
👍

File: pull_request_template.md [hunk 1]
```diff
@@ -10,5 +10,11 @@ I've checked that:
...
+* [ ] if adding a new materialized table, I edited the optimize table macro to be run in the background
+* [ ] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible
+* [ ] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
+* [ ] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')
 
-When you are ready for a review, tag duneanalytics/data-experience. We will re-open your forked pull request as an internal pull request. Then your spells will run in dbt and the logs will be avaiable in Github Actions DBT Slim CI. This job will only run the models and tests changed by your PR compared to the production project. 
+When you are ready for a review, tag duneanalytics/data-experience. We will re-open your forked pull request as an internal pull request. Then your spells will run in dbt and the logs will be avaiable in Github Actions DBT Slim CI. This job will only run the models and tests changed by your PR compared to the production project.
```
Comment (0xRobin) [inline pull_request_template.md:20]: FYI, outside users are not able to tag teams.
@duneanalytics/data-experience

File: pull_request_template.md [hunk 2]
```diff
@@ -10,5 +10,11 @@ I've checked that:
 * [ ] the filename is unique and ends with .sql
 * [ ] each sql file is a select statement and has only one view, table or function defined  
 * [ ] column names are `lowercase_snake_cased`
+* [ ] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
+* [ ] if adding a new model, I edited the alter table macro to display new database object (table or view) in UI explorer
+* [ ] if adding a new materialized table, I edited the optimize table macro to be run in the background
+* [ ] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible
+* [ ] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
+* [ ] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')
```
Comment (0xRobin) [inline pull_request_template.md:18]: maybe specify this is for `is_incremental()`

File: pull_request_template.md (hunk 1 above)
Comment (jeff-dude) [reply pull_request_template.md:20]: we're noticing that, thank you

**Pull request merged**